addons/sale/views/mail_activity_plan_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="mail_activity_plan_action_sale_order" model="ir.actions.act_window">
+            <field name="name">Sale Order Plans</field>
+            <field name="res_model">mail.activity.plan</field>
+            <field name="view_mode">tree,form</field>
```

Comment:
Don't forget priority to avoid default action taking this one

Reply:
Actually, when I do that, I get the error “Invalid field 'priority' on model 'ir.actions.act_window’” and indeed, there is no such field in that table. Is there case where the action is not explicitly specified ?